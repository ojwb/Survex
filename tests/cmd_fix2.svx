; pos=no warn=0 error=0
; Regression test for bug fixed in 1.4.17: We used to incorrectly warn:
; ./cmd_fix2.svx:2: warning: Unused fixed point "1"
*fix 1 0 0 0
*fix 2 0 0 0
2 3 1 0 0
*solve
1 4 1 90 0
