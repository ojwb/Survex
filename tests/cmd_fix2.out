
Removing trailing traverses...

Concatenating traverses...

Simplifying network...

Calculating network...

Calculating traverses...

Calculating trailing traverses...

Removing trailing traverses...

Concatenating traverses...

Simplifying network...

Calculating network...

Calculating traverses...

Calculating trailing traverses...

Calculating statistics...

Survey contains 4 survey stations, joined by 2 legs.
Total length of survey legs =    2.00m (   2.00m adjusted)
Total plan length of survey legs =    2.00m
Total vertical length of survey legs =    0.00m
Vertical range = 0.00m (from 3 at 0.00m to 3 at 0.00m)
North-South range = 1.00m (from 3 at 1.00m to 2 at 0.00m)
East-West range = 1.00m (from 4 at 1.00m to 3 at 0.00m)
